test/mbed_gt_test_filtered_test_list.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+"""
+mbed SDK
+Copyright (c) 2011-2015 ARM Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+from mbed_greentea import mbed_greentea_cli
+
+class GreenteaFilteredTestList(unittest.TestCase):
+
+    def setUp(self):
+        self.ctest_test_list = {'test1': '\\build\\test1.bin',
+                                'test2': '\\build\\test2.bin',
+                                'test3': '\\build\\test3.bin',
+                                'test4': '\\build\\test4.bin'}
+
+    def tearDown(self):
+        pass
+
+    def test_skip_test(self):
```

Comment:
Please write unit tests when parameters to this function are e.g. empty dictionaries and/or ```None```.